src/Entities/Entity.h

```diff
@@ -285,6 +284,14 @@ class cEntity
 
 	// tolua_end
 
+	/** Returns true if the entity is valid and ticking. Returns false if the entity is not ticking.
+	When the entity is not ticking, it is either scheduled for teleportation or for destruction.
+	Either way, using it is dangerous. */
+	bool IsTicking(void) const;
```

Comment:
I removed `IsDestroyed`. How do I deal with the LUA API here?

Perhaps I should export `IsTicking`, and make `IsDestroyed` a deprecated alias for `!IsTicking()`?.